arcade/examples/sprite_rotation.py

```diff
@@ -0,0 +1,76 @@
+"""
+Rotating Sprites Around Points
+
+A simple program to show how someone could implement sprites rotating around points.
+
+Artwork from https://kenney.nl
+"""
+import arcade
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
+
+SCREEN_TITLE = "Rotating Sprite Example"
+
+
+class RotatingSprite(arcade.Sprite):
+
+    def rotate_around_point(self, point, degrees, change_angle=True):
+        """
+        rotates the sprite around a defined point by the set amount of degrees
+        """
+
+        if change_angle:
+            # This is so the direction the sprite faces changes when rotating.
+            # It isn't necessary to have this. For example, you would want a rotating platform to always face upwards.
+            self.angle += degrees
+
+        # use arcade.rotate_point to rotate the sprites center x and y by the point.
+        self.center_x, self.center_y = arcade.rotate_point(self.center_x, self.center_y, point[0], point[1], degrees)
```

Comment:
To my understanding, there's no need to set these separately unless benchmarks proved this to be faster. Setting `position` should be enough. Setting both `center_x` and `center_y` will repeat the series of spatial hash resets that `position` does only once:
https://github.com/pythonarcade/arcade/blob/c1784151b6dace5104e4282362f43478a2d7e161/arcade/sprite.py#L667-L677
https://github.com/pythonarcade/arcade/blob/c1784151b6dace5104e4282362f43478a2d7e161/arcade/sprite.py#L684-L694

Reply:
Yeah, setting the position is faster. I just forgot.